integration_tests/src/lib.rs

```diff
@@ -398,11 +398,11 @@ mod test {
 
         // Verify contents are zero on init
         let buffer = Buffer::new(BLOCK_SIZE * 10);
-        volume
-            .read(Block::new(0, BLOCK_SIZE.trailing_zeros()), buffer.clone())
+        let buffer = volume
+            .read(Block::new(0, BLOCK_SIZE.trailing_zeros()), buffer)
             .await?;
 
-        assert_eq!(vec![0x00_u8; BLOCK_SIZE * 10], *buffer.as_vec().await);
+        assert_eq!(vec![0x00_u8; BLOCK_SIZE * 10], buffer.into_vec());
```

Comment:
Nit: use `&buffer[..]` here to match below?

Reply:
done in 919841b